server/chat/backend/agent/agent.py

```
@@ -484,15 +506,19 @@ def on_llm_end(self, response, run_id=None, **kwargs):
 
                 # Enable reasoning via OpenRouter's unified reasoning param
                 openrouter_model_kwargs = {}
+                is_background = getattr(state, "is_background", False)
                 if detected_provider == "openai":
                     from chat.backend.agent.providers.openai_provider import OpenAIProvider
                     native = model_name.split("/", 1)[-1] if "/" in model_name else model_name
                     if OpenAIProvider._supports_reasoning(native):
                         openrouter_model_kwargs["extra_body"] = {"reasoning": {"effort": "high"}}
                         logging.info(f"Enabled reasoning effort=high for {openrouter_model_name} via OpenRouter")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)